tests/controls/textctrltest.cpp

```diff
@@ -840,7 +844,7 @@ void TextCtrlTestCase::DoPositionToXYMultiLine(long style)
     text = wxS("123\nab\nX");
     m_text->SetValue(text);
 
-#if defined(__WXMSW__)
+#if defined(__WXMSW__) && !defined(__WXUNIVERSAL__)
```

Comment:
Minor, but I'd define some `wxHAS_2CHAR_NEWLINES` once and test for it rather than repeating the same test for `__WXMSW__ && !__WXUNIVERSAL__` several times.